packages/client/src/utils/referenceApi.ts

```diff
@@ -9,12 +9,12 @@
  * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
-import { IFormConfig } from '@client/forms'
-import { ILanguage } from '@client/i18n/reducer'
-import { ILocation } from '@client/offline/reducer'
-import { getToken } from '@client/utils/authUtils'
-import { Event, System } from '@client/utils/gateway'
-import { questionsTransformer } from '@client/forms/questionConfig'
+import {IFormConfig} from '@client/forms'
```

Comment:
@wilrona any idea why your editor wants to format the curly brackets like this?

Reply:
It is default conf of Intellij. but sometime, i take the time to fix it when eslint show me error.

Reply:
Is there a `prettier` plugin for Intellij? That's what we use for automatic formatting. I also recommend, for both David and yourself, to consider Visual Studio Code as an alternative (I know changing editors is a pain) as it has overall a better support for TypeScript and the tooling we're using. I also use Intellij IDEA for Scala & Clojure projects, but have noticed it's not as good for TypeScript